package.json

```diff
@@ -2,9 +2,11 @@
   "name": "webdav",
   "version": "1.0.1",
   "description": "WebDAV client for NodeJS",
-  "main": "source/index.js",
+  "main": "dist/index.js",
```

Comment:
I don't agree with changing the main to this though. I will most likely want the following:

 * Using in node: `require("webdav");`
 * Using in browser: `require("webdav/browser")` or `import createClient from "webdav/browser"`

I want the primary use case to remain node without transpilation. The browser transpiled copy can be it's own product.

Reply:
You don't have to change this, I'll do so when I merge.